test/interval.jl

```diff
@@ -136,46 +150,56 @@
             for i in 0:3
                 interval = Interval(a, b, Inclusivity(i))
                 cp = copy(interval)
-                lesser_val = Interval(a - unit, b + unit, Inclusivity(i))
+                lesser_val = Interval(a - unit, b - unit, Inclusivity(i))
```

Comment:
Change isn't required but I think it makes it clearer that this interval is always less than the original